src/std/str.mth

```diff
@@ -10,10 +10,10 @@ def(Str.num-bytes, Str -- Size, prim-str-num-bytes >Size)
 alias(Str.cat, prim-str-cat)
 def(Str.empty?, Str -- Bool, num-bytes 0=)
 
-||| Copy string bytes to second argument without adding 0 at end.
+||| Copy string bytes to second argument with-dataout adding 0 at end.
```

Comment:
```suggestion
||| Copy string bytes to second argument without adding 0 at end.
```

Reply:
oh no, shouldn't have find and replaced that :)